docs/getting_started/basic_config_file.rst

```diff
@@ -91,4 +91,23 @@ in order to e.g. execute a single test file inside the `suite_dir` using the CLI
 
     pykiso -c dummy.yaml -p test_suite_1.py
 
+
+To select test cases it is also possible to specify a regular expression which executes only
```

Comment:
maybe update the readme with this new cli parameter

Reply:
updated pykiso help output in readme